hammers/scripts/lease_stack_reaper.py

```diff
@@ -75,8 +78,8 @@ def find_stacked_leases(self, leases):
             else:
                 next_start_date = datetime.max
 
-            stacked_previous = (start_date - last_end_date).days < 1
-            stacked_next = (next_start_date - end_date).days < 1
+            stacked_previous = (start_date - last_end_date).days < 2
```

Comment:
maybe pull this "magic number" out to a constant `MIN_ALLOWED_STACK_INTERVAL_DAYS`.